src/modules/place/PlaceView.js

```diff
@@ -0,0 +1,266 @@
+    title: 'Place',
+    header: {
+      tintColor: theme.colors.text,
+      style: {
+        backgroundColor: theme.colors.navBar
+      }
+    }
+  }
+
+  static propTypes = {
+    office: PropTypes.object.isRequired,
+    loading: PropTypes.bool.isRequired,
+    place: PropTypes.object.isRequired,
+    back: PropTypes.func.isRequired,
+    officeStateActions: PropTypes.shape({
+      retryPlace: PropTypes.func.isRequired
+    }).isRequired
+  };
+
+  componentWillMount() {
+    //if this.prop.place doesn't contain the right data go back to home screen
+    if (!this.props.place.name) {
+      this.props.back();
+    }
+  }
+
+  onNextPress = () => {
+    this.props.officeStateActions.retryPlace(this.props.office, this.props.place);
+  }
+
+  buildPhotosURL = () => {
+    var size = this.props.place.photos.groups[0].items.length;
+    var photo = this.props.place.photos.groups[0].items[Utils.getRamdonNumberBetweenRange(size, 0)];
+    return (photo.prefix + '500x500' + photo.suffix);
+  }
+
+  getImage = () => {
+    return (this.props.place.photos && this.props.place.photos.count && this.props.place.photos.count > 0)
+      ? (<Image style={styles.image} source={{uri: this.buildPhotosURL()}}>{this.getRating()}</Image>)
+      : (<View style={styles.noImage}>{this.getRating()}</View>);
+  }
+
+  getPrice = () => {
+    var price = '';
+    if (this.props.place.price) {
+      for (let i = 0; i < this.props.place.price.tier; i++) {
+        price += this.props.place.price.currency;
+      }
+    }
+    return price;
+  }
+
+  getCategories = () => {
+    var categories = '';
+    if (this.props.place.categories) {
+      for (let i = 0; i < this.props.place.categories.length; i++) {
+        categories += this.props.place.categories[i].name + ', ';
+      }
+    }
+    return categories.slice(0, -2);
+  }
+
+  getAddress = () => {
+    var address = '';
+    if (this.props.place.location) {
+      address = (this.props.place.location.address)
+      ? this.props.place.location.address + ' '
+      : '';
+      address += (this.props.place.location.postalCode)
+      ? this.props.place.location.postalCode
+      : '';
+    }
+    return address;
+  }
+
+  getRatingStyles = () => {
+    return (this.props.place.ratingColor)
+    ? ({backgroundColor: '#' + this.props.place.ratingColor})
+    : '';
+  }
+
+  getRating = () => {
+    return (this.props.place.rating)
+      ? (<Image style={styles.gradient}
+              source={require('../../../assets/gradient.png')}>
+              <View style={[styles.ratingView, this.getRatingStyles()]}>
+                <Text style={styles.rating}>{this.props.place.rating}</Text>
+                <Text style={styles.ratingTotal}>/ 10</Text>
+              </View>
+            </Image>)
+      : (<Image style={styles.gradient} source={require('../../../assets/gradient.png')}/>);
+  }
+
+  getHours = () => {
+    return (this.props.place.hours)
+      ? (this.props.place.hours.status)
```

Comment:
Generally I haven't seen this style in javascript by wrapping everything in `(...)`. The code works perfectly fine without it. Maybe it would be good to get an eslint config for this? Leaving it up for others to comment as well.